specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -1683,6 +1691,56 @@
         "type": "object"
       }
     },
+    "CopyComputeScaleProperties": {
+      "description": "CopyComputeScale properties for managed integration runtime.",
+      "type": "object",
+      "properties": {
+        "dataIntegrationUnit": {
+          "description": "DIU number setting reserved for copy activity execution. Supported values are multiples of 4 in range 4-256.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 4
+        },
+        "timeToLive": {
+          "description": "Time to live (in minutes) setting of integration runtime which will execute copy activity.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 5
+        }
+      },
+      "additionalProperties": {
+        "type": "object"
+      }
+    },
+    "PipelineExternalComputeScaleProperties": {
+      "description": "PipelineExternalComputeScale properties for managed integration runtime.",
+      "type": "object",
+      "properties": {
+        "timeToLive": {
```

Comment:
>timeToLive [](http://example.com/codeflow?start=9&length=10)

Does it make sense to have the property as 'timeToLiveInMinutes'?

Reply:
The property name seems could not be changed since we are using it in service several years